### PR TITLE
Add cz.krupkat.Xpano

### PR DESCRIPTION
**Pull request by krupkat**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)* Submitted https://github.com/krupkat/xpano/pull/60

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: cz.krupkat.Xpano.yaml [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+        path: extra
+      - type: file
+        path: cz.krupkat.Xpano.metainfo.xml
+    config-opts:
+      - -DNFD_PORTAL=ON
+      - -DXPANO_EXTRA_LICENSES=licenses
+    post-install:
+      - install -D -m 644 -t $FLATPAK_DEST/share/metainfo ../cz.krupkat.Xpano.metainfo.xml
+
+finish-args:
```
Comment (hfiguiere) [inline cz.krupkat.Xpano.yaml:100]: finish-args are above modules usually

File: extra/licenses/opencv-license.txt
Comment (hfiguiere) [inline extra/licenses/opencv-license.txt:1]: this is unecessary

File: cz.krupkat.Xpano.yaml [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+  - name: xpano
+    buildsystem: cmake-ninja
+    builddir: true
+    sources: 
+      - type: git
+        url: https://github.com/krupkat/xpano
+        tag: "v0.9.0"
+        commit: 80a3d908cf0a683fb36761d5d56f49c914ac238d
+      - type: dir
+        path: extra
```
Comment (hfiguiere) [inline cz.krupkat.Xpano.yaml:91]:
```suggestion
```

**Comment by hfiguiere** [comment]
bot, build cz.krupkat.Xpano

**Comment by hfiguiere** [comment]
has the appdata file been upstreamed?

**Comment by krupkat** [comment]
> has the appdata file been upstreamed?

No, I haven't needed it in my repo yet. Should I add it there?

**Comment by krupkat** [comment]
Thanks for the quick review!
I removed the extra license files as you requested, instead I copy them from the repos (this is to show them in the gui).

**Comment by krupkat** [comment]
bot, build cz.krupkat.Xpano

**Comment by krupkat** [comment]
bot, build cz.krupkat.Xpano

**Comment by krupkat** [comment]
> > > has the appdata file been upstreamed?
> > 
> > 
> > No, I haven't needed it in my repo yet. Should I add it there?
> 
> it's not flatpak specific and future releases should ship it and install it.

ok, no problem
I have moved the file to upstream now

**Comment by krupkat** [comment]
bot, build cz.krupkat.Xpano

**Pull request closed without merging**